SkyDrive/SkyDrive.Tests/IntegrationTests/ControllerTests/InstructorControllerTests.cs

```diff
@@ -0,0 +1,173 @@
+﻿using FluentAssertions;
+using SkyDrive.Tests.FixtureCustomization.Attributes;
+using SkyDrive.ViewModels;
+using System.Net;
+using System.Net.Http.Json;
+
+namespace SkyDrive.Tests.IntegrationTests.ControllerTests
+{
+    public class InstructorControllerTests : IClassFixture<TestHttpClientFactory<Program>>
+    {
+        private readonly HttpClient _client;
+
+        public InstructorControllerTests(TestHttpClientFactory<Program> appFactory)
+        {
+            _client = appFactory.CreateClient();
+        }
+
+        [Fact]
+        public async Task GetAllInstructors_StatusCodeOK()
+        {
+            //Arrange
+            var response = await _client.GetAsync("Instructor");
+
+            //Act
+            var result = await response.Content.ReadFromJsonAsync<IEnumerable<InstructorViewModel>>();
+
+            //Assert
+            result.Should().NotBeNullOrEmpty();
+            response.StatusCode.Should().Be(HttpStatusCode.OK);
+        }
+
+        [Fact]
+        public async Task GetInstructorById_CorrectId_ReturnInstructorViewModel()
+        {
+            //Arrange
+            var instructorId = 1;
+
+            //Act
+            var response = await _client.GetAsync($"Instructor/{instructorId}");
+            var responseInstructorViewModel = await response.Content.ReadFromJsonAsync<InstructorViewModel>();
+
```

Comment:
Choose one style, please. Inside EventControllerTests.cs you use "ReadFromJsonAsync" construct inside the assert section. I guess it is better to do it like there, FIrst you parse results in the act section and then check it inside asserts.